bounties/npm/Superstatic/1/vulnerability.json

```diff
@@ -0,0 +1,52 @@
+{
+    "PackageVulnerabilityID": "1",
+    "DisclosureDate": "2020-08-07",
+    "AffectedVersionRange": "*",
+    "Summary": "Path Traversal",
+    "Discloser": {
```

Comment:
This object should be:

```
"Contributor": {
   "Discloser": "buggycoder-sys",
   "Fixer": ""
}
```

You only need to provide your GitHub username and leave the fixer empty.